src/mmw/js/src/modeling/models.js

```diff
@@ -9,6 +9,14 @@ var Backbone = require('../../shim/backbone'),
 var ModelPackageControlModel = Backbone.Model.extend({
     defaults: {
         name: ''
+    },
+
+    // Return true if this is an input control and false if it is a
+    // modification control.
+    isInputControl: function() {
```

Comment:
This is poorly named and will be address when I split modifications & input controls.